neo/ProtocolSettings.cs

```diff
@@ -10,7 +10,7 @@ public class ProtocolSettings
         public byte AddressVersion { get; }
         public string[] StandbyValidators { get; }
         public string[] SeedList { get; }
-        public uint SecondsPerBlock { get; }
+        public uint MillisecondsPerBlock { get; }
```

Comment:
I think that ulong makes more sense. In other programming languages Unix timestamp are all written using long (or ulong)

Reply:
Igor suggested as mandotory, uint goes to 4Bi and we were already in 1BI. With milliseconds it would not be possible.

Solved